File: health_monitoring_system_app/models/doctor.py
class Doctor:
    def __init__(self, id, name, surname, email, pesel, pwz):
        self.id = id
        self.name = name
        self.surname = surname
        self.email = email
        self.pesel = pesel
        self.pwz = pwz

    def to_dict(self):
        return {
            'id': self.id,
            'name': self.name,
            'surname': self.surname,
            'email': self.email,
            'pesel': self.pesel,
            'pwz': self.pwz
        }

File: HealthMonitoringSystemApplication/src/main/java/agh/edu/pl/healthmonitoringsystemapplication/databaseModels/Doctor.java
package agh.edu.pl.healthmonitoringsystemapplication.databaseModels;

import lombok.Builder;
import lombok.Getter;

import java.time.LocalDateTime;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;

@Table("doctor")
@Getter
@Builder
public class Doctor {
    @Id
    private Long id;
    private String name;
    private String surname;
    private String email;
    private String pesel;
    private String pwz;
    private LocalDateTime createdAt;

    public Doctor() {}

    public Doctor(Long id, String name, String surname, String email, String pesel, String pwz, LocalDateTime createdAt) {
        this.id = id;
        this.name = name;
        this.surname = surname;
        this.email = email;
        this.pesel = pesel;
        this.pwz = pwz;
        this.createdAt = createdAt;
    }
}
